bin/lsdcServer_amx.cmd

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -l
 export PROJDIR=/nsls2/software/mx/daq/
 export CONFIGDIR=${PROJDIR}bnlpx_config/
-export LSDCHOME=${PROJDIR}lsdc_amx
```

Comment:
Is the `${CONFIGDIR}daq_env_amx.txt` controlled by us or the user?  If it is the later anything we are going to rely on being what we set it to should probably be set _after_ the bl specific file.

Similarly, activating a conda env can set ENVS so maybe do that first?

Reply:
the entire ${CONFIGDIR} should not be editable by the user, we have full control of it.

I have found that it can be tricky to set ENVs, so would rather not touch them if this works